src/components/Table/body/styled/cell.js

```diff
@@ -14,7 +20,7 @@ const StyledCell = styled.th`
         outline: none;
 
         > div {
-            border-color: ${COLOR_BRAND};
+            border-color: ${props => props.brandMainColor};
```

Comment:
Currently, there is another PR with the changes in the table component, in order to not get conflicts, let's remove changes to Table in this PR